src/webviews/server.ts

```diff
@@ -40,7 +40,7 @@ export interface WebviewCompileOptions<
     D extends any[] = any[],
     S = any,
     O = any,
-    P extends any = any
+    P extends any = any // eslint-disable-line @typescript-eslint/no-unnecessary-type-constraint
```

Comment:
```suggestion
    P = any
```

Looks like I made a typo here.